package.json

```diff
@@ -80,5 +82,8 @@
         "typescript": "^5.0.4",
         "webpack": "^5.81.0",
         "webpack-cli": "^5.0.2"
-    }
+    },
+    "extensionDependencies": [
+        "tracecompass-community.vscode-trace-extension"
```

Comment:
Use new publisher

Reply:
The new publisher of the vscode-trace-extension is: `eclipse-cdt` which was changed in https://github.com/eclipse-cdt-cloud/vscode-trace-extension/pull/191